test/unit/factorization/mc/test_compute_t_factor.cpp

```diff
@@ -327,14 +368,19 @@ TYPED_TEST(ComputeTFactorTestMC, CorrectnessDistributed) {
 
       is_orthogonal(h_expected);
 
-      common::Pipeline<comm::CommunicatorGrid> serial_comm(comm_grid);
+      common::Pipeline<comm::Communicator> serial_comm(comm_grid.colCommunicator());
 
-      Matrix<TypeParam, Device::CPU> t_output({k, k}, block_size);
+      const auto max_k = std::min(mb, nb);
+      Matrix<TypeParam, Device::CPU> t_output({max_k, max_k}, block_size);
       const LocalTileIndex t_idx(0, 0);
 
       dlaf::factorization::internal::computeTFactor<Backend::MC>(k, v_input, v_start, taus_input,
                                                                  t_output(t_idx), serial_comm);
 
+      // No reflectors, so nothing to check
```

Comment:
I would check it.

Reply:
`computeTFactor` can alter just the tile used for storing the TFactor, since the other parameters are readonly, and in this case it means that the original tile is exactly as it was before the call (no reflectors, no tile for tfactor).

Is it what you'd like to check or were you thinking about something more?

Reply:
If nothing can change no check is needed.